contrib.json

```diff
@@ -336,21 +336,22 @@
         [ "git branch -D {{pr.user.login}}-{{pr.head.ref}}",                   "Delete the local branch used for merging" ]
       ]
     },
-    "accept-as-is": {
+     "accept-as-is": {
       "steps": [
         { "prompt": "text", "id": "prNum",                             "desc": "What is the the pull request number?" },
         { "get": "{{meta.urls.repo_api}}/pulls/{{prNum}}",             "desc": "Get the PR information", "id": "pr" },
         { "get": "{{meta.urls.repo_api}}/pulls/{{prNum}}/commits",     "desc": "Get the PR commits to access author info", "id": "prCommits" },
         [ "git checkout -b {{pr.user.login}}-{{pr.head.ref}} {{pr.base.ref}}", "Create a new branch for merging the changes" ],
-        [ "git pull --rebase {{pr.head.repo.ssh_url}}",                        "Merge and rebase the changes"],
-        [ "grunt test",                                                        "Run tests to make sure they still pass" ],
+        [ "git pull {{pr.head.repo.ssh_url}} {{pr.head.ref}}",                 "Merge in the changes"],
+        [ "git rebase {{pr.base.ref}}",                                        "Rebase against the latest changes in the base branch"],
```

Comment:
I was trying to ensure the change was applied cleanly on top of HEAD, not merged. My understanding is this after-the-fact rebase accomplishes that. You don't think so?